apps/api/v2/src/modules/oauth-clients/controllers/oauth-client-users/outputs/create-managed-user.output.ts

```diff
@@ -34,4 +34,6 @@ export class CreateManagedUserOutput {
   @ValidateNested()
   @Type(() => CreateManagedUserData)
   data!: CreateManagedUserData;
+
+  error?: Error;
```

Comment:
Is this error object the one that would be sent back via `ConflictException`?

Reply:
This is just a general Error and I think that ConflictException extends Error class